src/JSInjection/Modal/SaveToZeeguu.js

```diff
@@ -1,28 +1,40 @@
 import {
-  StyledSmallButtonBlue,
+  StyledSmallButton,
   StyledSmallDisabledButton,
 } from "./Buttons.styles";
+import { useState } from "react";
 import {EXTENSION_SOURCE} from "../constants";
+import BeenhereIcon from '@mui/icons-material/Beenhere';
+import BeenhereOutlinedIcon from '@mui/icons-material/BeenhereOutlined';
+import Tooltip from '@mui/material/Tooltip';
+
 
-export default function SaveToZeeguu({ api, articleId, setPersonalCopySaved, personalCopySaved}) {
 
+export default function SaveToZeeguu({ api, articleId, setPersonalCopySaved, personalCopySaved}) {
   function handlePostCopy() {
     let article = { article_id: articleId };
     api.makePersonalCopy(article, (message) => console.log(message));
     api.logReaderActivity(api.PERSONAL_COPY, articleId, "", EXTENSION_SOURCE);
     setPersonalCopySaved(true);
   }
+  const [isHovered, setIsHovered] = useState(false);
 
   return (
     <>
       {personalCopySaved ? (
-        <StyledSmallDisabledButton>
-          Saved on zeeguu.org
-        </StyledSmallDisabledButton>
+        <Tooltip title="Saved to your Zeeguu account" arrow>
+          <StyledSmallDisabledButton>
+            <BeenhereIcon fontSize="large"/> <br/>
+            Saved
+          </StyledSmallDisabledButton>
+        </Tooltip>
       ) : (
-        <StyledSmallButtonBlue onClick={handlePostCopy}>
-          Save article to zeeguu.org
-        </StyledSmallButtonBlue>
+        <StyledSmallButton onClick={handlePostCopy}
```

Comment:
It would make sense to have the button together with the hover behavior as it's own component maybe?